lib/jit/generation/elementwise_2d.cpp

```diff
@@ -118,6 +118,7 @@ void elementwise_2d::enqueue(driver::CommandQueue & /*queue*/, driver::Program c
   symbolic::set_arguments(expressions, kernel, current_arg);
 
   control.execution_options().enqueue(program.context(), kernel, global, local);
+  driver::dispatch::clFinish(control.execution_options().queue(expressions.context()).handle().cl());
```

Comment:
We can't add clFinish here. If it could fix a test failures, you need to find out the root cause of that failure. In this function, we only need to enqueue the corresponding kernel. It's the caller's responsibility to make sure some synchronization between host and the device.